app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
+        return
+    command, source, target = parts
+    if parts[0] != "mv" or source == target:
```

Comment:
Replace `parts[0]` with the `command` variable, you already performed unpacking. Besides that, check all three conditions in one if condition, there in no need to split them into two if conditions. You could use `any` for that.